shared-libs/cht-datasource/src/remote/person.ts

```diff
@@ -19,4 +19,13 @@ export namespace v1 {
     identifier.uuid,
     { with_lineage: 'true' }
   );
+
+  /** @internal */
+  export const getPage = (remoteContext: RemoteDataContext) => (
+    limit: number,
+    skip: number
+  ): Promise<null> => getPerson(remoteContext)(
```

Comment:
For this function, I don't think we want to reuse the `getPerson`/`getResource` code.  I guess there are some natural differences between getting a specific entity vs getting a list of entities and the most straightforward way to handle this seems like it would be by adding a new function:

```ts
/** @internal */
export const getResources = (context: RemoteDataContext, path: string) => async <T>(
queryParams?: Record<string, string>
): Promise<T> => {
const params = new URLSearchParams(queryParams).toString();
try {
const response = await fetch(`${context.url}/${path}?${params}`);
if (!response.ok) {
throw new Error(response.statusText);
}

    return (await response.json()) as T;
} catch (error) {
logger.error(`Failed to fetch resources from ${context.url}/${path} with params: ${params}`, error);
throw error;
}
};
```

The most important difference is that there is no special handling for a `404`.